jans-core/cache/src/main/java/io/jans/service/cache/NativePersistenceCacheProvider.java

```diff
@@ -82,6 +82,9 @@ public void create() {
             // CouchbaseEntryManagerFactory.PERSISTENCE_TYPE
             skipRemoveBeforePut = "couchbase".equals(persistenceType);
             attemptUpdateBeforeInsert = "sql".equals(persistenceType);
+            if (cacheConfiguration.getNativePersistenceConfiguration().isDisableAttemptUpdateBeforeInsert()) {
```

Comment:
I guess we don't need explicit configuration for it. Existing `attemptUpdateBeforeInsert = "sql".equals(persistenceType);` is enough. With this we already updating entry (for all further calls except initial one).

@yurem is it what you meant in https://github.com/JanssenProject/jans/issues/207 ?

Reply:
Actually, I talked with him before doing these changes, but let's see Yure's feedback in case I misunderstood something.

Reply:
yes, for #207 we already updated code. But this new property will allow us to check at runtime which approach is better for specific DB MySQL/Postgress/etc

Reply:
Agreed, yes, it can be useful.